selenium/che-selenium-test/src/main/java/org/eclipse/che/selenium/core/constant/TestBuildConstants.java

```diff
@@ -15,6 +15,7 @@
   public static final String BUILD_SUCCESS = "[INFO] BUILD SUCCESS";
   public static final String BUILD_FAILED = "[INFO] BUILD FAILURE";
   public static final String SERVER_STARTUP_IN = "Server startup in";
+  public static final String LISTENING_AT_ADDRESS = "Listening for transport dt_socket at address";
   public static final String LISTENING_AT_ADDRESS_8000 =
```

Comment:
We can simplify it as follow:
`LISTENING_AT_ADDRESS_8000 = LISTENING_AT_ADDRESS + ": 8000";`